apps/src/templates/sectionProgressV2/SectionProgressSelector.jsx

```diff
@@ -100,11 +101,14 @@ function SectionProgressSelector({
 
   const includeModalIfAvailable = () => {
     const disableModal = DCDO.get('disable-try-new-progress-view-modal', false);
-    if (!disableModal) {
+    if (disableModal || hasJustToggledViews) {
+      return;
+    }
+    if (!hasSeenProgressTableInvite) {
       return (
         <InviteToV2ProgressModal
           sectionId={sectionId}
-          setHasJustSwitchedToV2={setHasJustSwitchedToV2}
+          setHasJustSwitchedToV2={setHasJustToggledViews}
```

Comment:
I did keep the prop name for `InviteToV2ProgressModal` the same since I think it made the code for InviteToV2ProgressModal slightly more readable.